src/hooks/common/useTimer.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+
+export const useTimer = () => {
+  const [totalSecond, setTotalSecond] = useState(0);
```

Comment:
p3;
state에 타입 작성 부탁드립니다!

```suggestion
  const [totalSecond, setTotalSecond] = useState<number>(0);
```

Reply:
자꾸 useState에 타입 넣는것 까먹네요 확인해주셔서 감사합니다!! 타입 적용했습니다

작업 커밋 : 1c60d4fba7663e9b21fc83ee24e28fcf97f059f5